### PR TITLE
Feature/8 create many board tables

### LINKED ISSUE #8: Create many board (tables)
### Create board (Table) with columns as a separate React Component
* Each board can hold infinity number of columns.
* Each board must be available only for specific users

### PR DESCRIPTION
**Pull request by StoyanTinchev**
### Added: 
* Creation of boards
* Creation of columns for specific board
* Store everything on local storage

### Edited:
* Use props to get users massive

### TODO:
* function getBoard is implemented three times in different places and can be more generic

### CODE REVIEW

File: src/components/Board/Board.js [hunk 1]
```diff
@@ -1,15 +1,62 @@
-import React from 'react';
-import PropTypes from 'prop-types';
+import React, {useState} from 'react';
```
Comment (AntonioK180) [inline src/components/Board/Board.js:1]: not sure about this import

File: src/components/CreateBoard/CreateBoard.js [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+        console.log(boardName);
+        console.log(board);
+        if (board) {
+            user.active_board = board.id;
+            console.log(board.id);
+            localStorage.setItem("users", JSON.stringify(props.users));
+            return;
+        }
+
+        const id = uuidv4();
```
Comment (AntonioK180) [inline src/components/CreateBoard/CreateBoard.js]: don't need this variable

**Review by AntonioK180 — CHANGES_REQUESTED** [review]
Remove unnecessary id variable.

**Review by AntonioK180 — APPROVED** [review]

File: src/components/Board/Board.js (hunk 1 above)
Comment (StoyanTinchev) [reply src/components/Board/Board.js:1]: Fixed

File: src/components/CreateBoard/CreateBoard.js (hunk 2 above)
Comment (StoyanTinchev) [reply src/components/CreateBoard/CreateBoard.js]: Fixed

**Pull request merged**